general/src/protocol/contexts.rs

```diff
@@ -210,6 +210,12 @@ impl BrowserContext {
     }
 }
 
+/// Operation type such as `db.statement` for database queries or `http` for external HTTP calls.
+/// Tries to follow OpenCensus/OpenTracing's span types.
+///
+/// TODO typing ( union of predefined types and custom 'string' types)
```

Comment:
Probably fine if it’s not typed out, just like breadcrumbs. The only place where we would benefit from this is Sentry-types, probably.

Reply:
I'll remove the TODO. IMO the type alias is still useful for documentation